core/src/docs/rfcs/3574_concurrent_stat_in_list.md

```diff
@@ -0,0 +1,62 @@
+- Proposal Name: `concurrent_stat_in_list`
+- Start Date: 2023-09-16
```

Comment:
date should be `2023-11-13`

Reply:
Done. (I thought the date is the data you brought for the problem. 😅)